libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/ActiveGuidanceScreen.kt

```diff
@@ -71,13 +75,15 @@ class ActiveGuidanceScreen(
         }
     }
 
+    private var styleLoadedListener: OnStyleLoadedListener? = null
+
     private val surfaceListener = object : MapboxCarMapObserver {
 
         override fun onAttached(mapboxCarMapSurface: MapboxCarMapSurface) {
             super.onAttached(mapboxCarMapSurface)
             logAndroidAuto("ActiveGuidanceScreen loaded")
-            mapboxCarMapSurface.mapSurface.getMapboxMap().getStyle { style ->
-                placesLayerUtil.initializePlacesListOnMapLayer(style, carContext.resources)
+            styleLoadedListener = mapboxCarMapSurface.handleStyleOnAttached {
```

Comment:
Same here https://github.com/mapbox/mapbox-navigation-android/pull/5853/files#r888158043

This one is actually bad because it is registering multiple `routeObservers`